### PR TITLE
Apply code style

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

File: upload/system/library/document.php [hunk 1]
```diff
@@ -40,7 +42,6 @@ public function getTitle(): string {
 	 * setDescription
 	 *
 	 * @param string $description
-	 *
 	 * @param void
```
Comment (AJenbo) [inline upload/system/library/document.php:45]:
```suggestion
	 *
	 * @return void
```

File: upload/catalog/model/extension/payment/pilibaba.php [hunk 2]
```diff
@@ -67,7 +67,6 @@ public function addPilibabaOrder(array $response_data): void {
 	 * getConsumerInfo
 	 *
 	 * @param int $order_id
-	 *
 	 * @param array
```
Comment (AJenbo) [inline upload/catalog/model/extension/payment/pilibaba.php:70]:
```suggestion
	 *
	 * @return array
```

**Comment by TheCartpenter** [comment]
> @AJenbo how you make that changes so fast? :)

That's what trained people do, 80-90 words per minute.

**Comment by TheCartpenter** [comment]
Still have the green marks. No issues but we need to make sure the ```null``` validations are after the variables; not before. It's normally how it's done in the core (and, in this case, extensions as well).

**Comment by AJenbo** [comment]
I'm even just lazily doing it from bed on a slow laptop :D

**Comment by condor2** [comment]
> > @AJenbo how you make that changes so fast? :)
> 
> https://github.com/opencart/opencart/wiki/Coding-standards#cs-fixer

Interesting. Tks
Using this will make changes 12k changes in 10-16 mins :rofl:

**Pull request merged**